### ISSUE TITLE
dotnet ef database update -s ../API causes "No such host is known" error

### ISSUE THREAD

**Issue by smann004** [madslundt/knowledge-quiz-api#1]
I have followed the steps in "Setting up application" but am unable to run the migrations. Is PostgreSQL required for this project? It seems the stack trace shows an error originating from Npgsql which, as far as I can tell, is for PostgreSQL.
Thanks

**Comment by madslundt** [comment]
Yes the database that is used is Postgresql. You have to enable uuidv4 in the database before running the migrations.
